service/awsconfig/drainer_framework.go

```diff
@@ -150,8 +150,8 @@ func newDrainerResourceRouter(config DrainerFrameworkConfig) (*framework.Resourc
 	var v8ResourceSet *framework.ResourceSet
 	{
 		c := v8.DrainerResourceSetConfig{
-			GuestAWSClients: awsClients,
-			Logger:          config.Logger,
+			AWS:    awsClients,
```

Comment:
When I was hacking on aws-operator I noticed distinguishing between guest and host AWS clients is hard, therefore I think it's good to add `Guest` or `Host` to AWS client names.

Reply:
I agree this is hard currently and knowing if the client is guest or host is very useful.

Reply:
I found it odd to differentiate something where there is no potential of collision. This is for the lifecycle resource where there is no host client magic. Do you want to be explicit even though there is only the guest client?

Reply:
That's fair its only the CloudFormation resource that uses both. For me having guest in the name isn't a problem but if you want it to be shorter I'm fine with it.

Reply:
Same.

Reply:
Ok going with `AWS`. Thanks.